src/train_classifier_g.py

```diff
@@ -0,0 +1,31 @@
+from suite import Suite
+import pandas as pd
+from classifiers import *
+from pathlib import Path
+from os.path import join
+import os
+path = Path(os.path.abspath("."))
+
+def train_model():
+  s = Suite()
+
+  print("Loading training data.")
+  train_dat = pd.read_csv(join(path, "src/data/training_comments.csv"))
+  s.set_train_set(train_dat)
+  test_dat = pd.read_csv(join(path, "src/data/test_comments.csv"))
+  s.set_test_set(test_dat)
+  s.set_model(svm_model)
+
+  s.set_ratios([2])
+  s.add_parameter("C", [.05])
+  s.add_parameter("gamma", [2])
+
+  s.features = ["perspective_score", "stanford_polite"]
```

Comment:
It's worth revisiting this. As we're labeling more data now, we're finding that the generalizability of the original classifier may be affected. Perhaps going back and retraining with more features, from scratch, would work better?